pkgs/development/libraries/aspell/default.nix

```diff
@@ -50,6 +44,7 @@ stdenv.mkDerivation rec {
   # both include those files.
   postInstall = ''
     cp ${devaMapsSource}/u-deva.{cmap,cset} $out/lib/aspell/
+    patchShebangs $out/bin/aspell-import
```

Comment:
It's a Perl script, so I guess `perl` should rather be in `buildInputs`

And with `perl` in build inputs, the shebang patching should be done automatically and you would'nt need this line

Reply:
done. Thanks.